hardhat/hardhat.config.js

```diff
@@ -0,0 +1,149 @@
+require("@nomiclabs/hardhat-waffle");
+const fs = require('fs');
+
+// This is a sample Hardhat task. To learn how to create your own go to
+// https://hardhat.org/guides/create-task.html
+task("accounts", "Prints the list of accounts", async (taskArgs, hre) => {
+  const accounts = await hre.ethers.getSigners();
+
+  for (const account of accounts) {
+    console.log(account.address);
+  }
+});
+
+function loadconf() {
+  try {
+    const conf = fs.readFileSync('state.conf');
+    return JSON.parse(conf);
+  }
+  catch {
+    return {
+      mainbridge: {},
+      subbridge: {},
+    };
+  }
+}
+
+function saveconf(conf) {
+  try {
+    fs.writeFileSync('state.conf', JSON.stringify(conf));
+  }
+  catch(err) {
+    console.log('writing to state.conf failed', err);
+    return;
+  }
+}
+
+task("deploy", "Deploy bridge and token contracts to both EN and SCN", async (taskArgs, hre) => {
+  const conf = loadconf();
+
+  const Bridge = await hre.ethers.getContractFactory("Bridge");
+  const bridge = await Bridge.deploy(true);
+  await bridge.deployed();
+
+  const Token = await hre.ethers.getContractFactory("ServiceChainToken");
+  const token = await Token.deploy(bridge.address);
+  await token.deployed(bridge.address);
+
+  await token.addMinter(bridge.address);
+
+  await bridge.registerOperator(hre.network.config.operator);
+
+  if (hre.network.name == 'mainbridge') {
+    conf.mainbridge.bridge = bridge.address;
+    conf.mainbridge.token = token.address;
+  }
+  else if (hre.network.name == 'subbridge') {
+    conf.subbridge.bridge = bridge.address;
+    conf.subbridge.token = token.address;
+  }
+  saveconf(conf);
+});
+
+task("regtoken", "Register operator and token to bridge", async (taskArgs, hre) => {
+  const conf = loadconf();
+  const c = conf[hre.network.name];
+  const Bridge = await hre.ethers.getContractFactory("Bridge");
+  const bridge = await Bridge.attach(c.bridge);
+
+  if (hre.network.name == 'mainbridge') {
+    await bridge.registerToken(c.token, conf.subbridge.token);
+  }
+  else {
+    await bridge.registerToken(c.token, conf.mainbridge.token);
+  }
+  await bridge.transferOwnership(hre.network.config.operator);
+
+  if (hre.network.name == 'subbridge') {
```

Comment:
The original value transfer examples use the `subbridge` RPC to perform the actions done by those commands, thus the user is not required to run any commands. Since we already require `eth` RPC to be enabled for `kscn`, how about asking to enabling one more RPC namespace, and run `subbridge` RPCs instead of asking the user to run some commands?

Reply:
Good idea. I will add a commit in the near future.

Reply:
done